### PR TITLE
yet another fix to the nftables script

### PR DESCRIPTION
**Pull request by aojea**
bash and quotes things

```
apiVersion: kubeproxy.config.k8s.io/v1alpha1
conntrack:
  maxPerCore: 0
featureGates:
  NFTablesProxyMode:true: false
iptables:
  minSyncPeriod: 1s
```

Change-Id: I1c1c2911514bb87c5a0f57295b4e480e06472cfa

### CODE REVIEW

**Comment by aojea** [comment]
/assign @danwinship 

This things always take several iterations :D

File: experiment/kind-nftables-e2e.sh [hunk 1]
```diff
@@ -107,7 +107,7 @@ create_cluster() {
   fi
 
   # JSON map injected into featureGates config
-  feature_gates="{"NFTablesProxyMode":true}"
+  feature_gates='{"NFTablesProxyMode":"true"}'
```
Comment (danwinship) [inline experiment/kind-nftables-e2e.sh:110]: pretty sure you want

```suggestion
  feature_gates='{"NFTablesProxyMode":true}'
```

Comment (aojea) [reply experiment/kind-nftables-e2e.sh:110]: 👍

**Comment by aojea** [comment]
all green now https://testgrid.k8s.io/sig-network-kind

**Pull request merged**